src/command/subgraph/mod.rs

```diff
@@ -17,13 +18,17 @@ pub struct Subgraph {
 
 #[derive(Debug, Serialize, StructOpt)]
 pub enum Command {
-    /// Push an implementing service schema from a local file
-    Push(push::Push),
+    /// Validate changes to an implementing service
```

Comment:
```suggestion
    /// Checkchanges to an implementing service
```

Maybe we could use the `check` vernacular all over instead of validate? Although that's a tiny nit